app/bundles/MessengerBundle/Translations/en_US/messages.ini

```diff
@@ -1,23 +1,14 @@
-mautic.config.tab.messengerconfig="Messenger Settings"
-mautic.messenger.config.enabled="Do you want to send messages (emails, sms, push) via a queue?"
-mautic.messenger.config.enabled.true="Yes"
-mautic.messenger.config.enabled.false="No, I want to send messages immediately"
-mautic.messenger.config.enabled.tooltip="Messages are sent via a queue so that the system doesn't slow down. This means that if you have a lot of messages to send, it could take a while to send them all."
-mautic.messenger.config.transport="Queue transport type"
-mautic.messenger.config.transport.tooltip=""
-mautic.messenger.config.host="Host"
-mautic.messenger.config.host.tooltip="Host"
-mautic.messenger.config.port="Port"
-mautic.messenger.config.port.tooltip="Port"
-mautic.messenger.config.user="Username"
-mautic.messenger.config.user.tooltip="Username"
-mautic.messenger.config.password="Password"
-mautic.messenger.config.password.tooltip="Password"
+mautic.config.tab.messengerconfig="Queue Settings"
 mautic.messenger.config.retry_strategy="Retry strategy"
 mautic.messenger.config.retry_strategy.max_retries="Max retries"
 mautic.messenger.config.retry_strategy.delay="Delay"
 mautic.messenger.config.retry_strategy.multiplier="Multiplier"
 mautic.messenger.config.retry_strategy.max_delay="Max delay"
-mautic.messenger.config.transport.doctrine="Database Queue"
-
-
+mautic.messenger.config.dns_help_general="Queuing is not enabled by default (scheme is set to 'sync'). If you want to start using a queue, please follow the documentation on <a href="%link%">%link%</a>."
+mautic.messenger.config.dns_help_retry_strategy="When processing of a message fails, the message is sent back to the queue for another try. You can tweak this behaviour in the following section. See the documentation on <a href="%link%">%link%</a> for more details."
+mautic.messenger.config.dns_help_failed="If a message fails all its retries, it is discarded by default. To avoid this happening, you can optionally configure a queue for failures. For more details see the documentation on  <a href="%link%">%link%</a>."
```

Comment:
```suggestion
mautic.messenger.config.dns_help_failed="If a message fails all its retries, it's discarded by default. To avoid this happening, you can optionally configure a queue for failures. For more details see the documentation on  <a href="%link%">%link%</a>."
```